framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/OpenShiftScenario.java

```diff
@@ -246,9 +248,27 @@ public void afterScenarioFinished(T deploymentScenario) {
     @Override
     public MavenRepositoryDeployment getMavenRepositoryDeployment() {
         return externalDeployments.stream().filter(deployment -> ExternalDeploymentID.MAVEN_REPOSITORY.equals(deployment.getKey()))
-                                           .map(ExternalDeployment::getDeploymentInformation)
-                                           .map(deployment -> (MavenRepositoryDeployment) deployment)
-                                           .findAny()
-                                           .orElseThrow(() -> new RuntimeException("Maven repository deployment not found."));
+                .map(ExternalDeployment::getDeploymentInformation)
+                .map(deployment -> (MavenRepositoryDeployment) deployment)
+                .findAny()
+                .orElseThrow(() -> new RuntimeException("Maven repository deployment not found."));
+    }
+
+    private void deploySecretConfig() {
+        logger.info("Creating generally used secret from " + OpenShiftTemplate.SECRET.getTemplateUrl().toString());
+        Map<String, String> secretConfig = new HashMap<>();
+        secretConfig.put(OpenShiftConstants.SECRET_NAME, OpenShiftConstants.getKieApplicationSecretName());
+        secretConfig.put(OpenShiftConstants.CREDENTIALS_SECRET, DeploymentConstants.getAppCredentialsSecretName());
```

Comment:
Why do you pass CREDENTIALS_SECRET here?

Reply:
In the method "deploySecretAppUser", we create the secret named as "X" and using the properties "CREDENTIALS_SECRET" we tell BC to use the secret "X". 
Does it make sense to you?

Reply:
Yes, but AFAIK the CREDENTIALS_SECRET here is used just for creation of secret template (below). It is not related to Workbench (unless I missed something).

Reply:
I think you're right. I will remove it. Thanks for spotting this!